app/[locale]/page.tsx

```diff
@@ -8,8 +9,7 @@ export default function Home({
 }) {
   return (
     <main className="w-full">
-      <Notifications />
-      <DirectorsCorner locale={locale} />
+      <Events />
```

Comment:
Don't remove the other components.
```suggestion
      <Notifications />
      <Events /> 	
      <DirectorsCorner locale={locale} />
```